src/rgw/rgw_lc.cc

```diff
@@ -2678,23 +2678,18 @@ int RGWLC::set_bucket_config(rgw::sal::Bucket* bucket,
   return ret;
 }
 
-int RGWLC::remove_bucket_config(rgw::sal::Bucket* bucket,
-                                const rgw::sal::Attrs& bucket_attrs,
-				bool merge_attrs)
+int RGWLC::remove_bucket_config(rgw::sal::Bucket* bucket)
```

Comment:
we still need the `bool` argument for callers that want to avoid writing the bucket instance metadata. this is `false` when the bucket is being deleted anyway, or when the metadata sync is already overwriting the bucket instance with new attrs

opened https://github.com/ceph/ceph/pull/62980 with the adjustments